static/src/javascripts/lib/raven.js

```diff
@@ -33,6 +33,8 @@ const sentryOptions = {
         'Network request failed',
         'This video is no longer available.',
         'UnknownError',
+        'TypeError: Failed to fetch',
```

Comment:
The CMP depends on a network work to be able to run. If it doesn't run our website is not compliant with privacy laws. Adding these items to the ignore list means we won't be able to detect outages in the CMP.

Reply:
You're right, it's important that we make sure that we're alerted in the event that this endpoint were to go down but I'm not sure Sentry if the right tool for that task. Is it worth raising a separate task to ensure we're tracking this (and maybe other) key endpoints in case they go down?

It's also worth noting that we're already filtering out several other variations of network failure errors so Sentry is already a flawed solution in terms of checking if endpoints are up or not.

Reply:
As far as I know this is the only endpoint that has the potential for such serious consequences in case it goes down so maybe it's not worth setting up something to monitor endpoints in general. This endpoint is will stop being used within the next 2 months as we upgrade our CMP to use Sourcepoint's integration instead.
Two options seem to available to us:
- We merge this PR and hope for the best until the CMP no longer fetches from that endpoint (will happen in the next 2 months). This is a link that is accessed worldwide so if it has issues (even if localised) then everyone has issues, which means it's more likely to be detected and fixed quickly.
- We come up with a solution to check if the endpoint is up before merging this PR. This solution would most likely not be able to catch localised outages as it would check the endpoint is up always from the same location.

Let me talk to comm dev tomorrow after planning to try to reach a consensus and I'll leave an update on this PR. Is that ok?

Reply:
Sure, sounds good. Thanks for looking into this.

If we just wanted a stopgap solution for 2 months then maybe https://uptimerobot.com/? It's free for checks every 5 minutes so no need to worry about billing.

Reply:
I think we'd be better off catching this error in code and sampling it instead of blanket ignoring potentially important fetching errors?

Reply:
We talked about it as a team and the consensus is we can go ahead and merge this as far as we are concerned. We are checking the availability of this endpoint at runtime and that will be sufficient until we retire it in the near future. So if the dotcom team is happy go ahead and ship it @oliverlloyd 👍

Reply:
Thanks @ripecosta for taking the time to think this one through